lib/audited/audit.rb

```diff
@@ -140,6 +134,18 @@ def set_audit_user
     def set_request_uuid
       self.request_uuid ||= SecureRandom.uuid
     end
+
+    # Returns a hash of the changed values. "new" for new values, "old" for old values
+    def old_or_new_attributes(type)
```

Comment:
Not a fan of this.  This "extraction" refactor adds more complexity in order to dry up just a few lines.